core/client/app/components/gh-profile-image.js

```diff
@@ -28,12 +29,25 @@ export default Ember.Component.extend({
         return `background-image: url(${url})`.htmlSafe();
     }),
 
+    trySetValidEmail: function () {
+        var email = this.get('email');
+        if (validator.isEmail(email)) {
+            this.set('validEmail', email);
+        } else {
+            this.set('validEmail', '');
+        }
+    },
+
+    onEmailChange: Ember.observer('email', function () {
+        Ember.run.debounce(this, 'trySetValidEmail', 700);
+    }),
+
     imageBackground: Ember.computed('email', 'size', function () {
```

Comment:
just realized I didn't switch back to observing 'validEmail' in the component. will not push code at midnight anymore :smile: